app/client/styles/main.styl

```diff
@@ -48,6 +48,7 @@ html, body
   background-position: center
   min-height: 100%
   color: $color-black
+  visibility: hidden
```

Comment:
because the file upload pkg has a flash fallback and on load you get a little flash of the flash puzzle piece and the screen is all fucked up before the mainlayout loads